packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -740,7 +742,43 @@ async function formNestedStack(
   const { amplifyMeta } = projectDetails;
   let authResourceName: string;
 
-  const { NetworkStackS3Url } = amplifyMeta.providers[constants.ProviderName];
+  const { NetworkStackS3Url, AuthTriggerTemplateURL } = amplifyMeta.providers[constants.ProviderName];
+
+  if (AuthTriggerTemplateURL !== '') {
```

Comment:
As the code is currently written, I think this will be `undefined` as well when the FF is off.

Reply:
Nice catch, can also add a check for undefined too

Reply:
It might be easiest to just check `if (AuthTriggerTemplateURL)` since `''` and `undefined` are both falsy. If you want to get really strict, you could check `if (typeof AuthTriggerTemplateURL === 'string' && AuthTriggerTemplateURL.length > 0)`, but that's probably overkill.